src/lib/Components/ReadMore.tsx

```diff
@@ -0,0 +1,121 @@
+import { Flex, Sans, Serif } from "@artsy/palette"
+import { plainTextFromTree } from "lib/utils/plainTextFromTree"
+import { defaultRules, renderMarkdown } from "lib/utils/renderMarkdown"
+import _ from "lodash"
+import React, { useState } from "react"
+import { Text } from "react-native"
+import { LinkText } from "./Text/LinkText"
+
+interface Props {
+  content: string
+  maxChars: number
+}
+
+const rules = {
+  ...defaultRules,
+  paragraph: {
+    ...defaultRules.paragraph,
+    react: (node, output, state) => {
+      return (
+        <Serif size="3t" color="black60" key={state.key}>
+          {output(node.content, state)}
+        </Serif>
+      )
+    },
+  },
+}
+
+export const ReadMore = React.memo(({ content, maxChars }: Props) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  const root = renderMarkdown(content, rules)
+
+  const plainTextVersion = plainTextFromTree(root)
```

Comment:
@ds300 I pulled this into a separate util because it could be used elsewhere, but also it didn't seem worth muddying up the existing `truncate` component.

Not tied to this implementation at all, though.